### ISSUE TITLE
Suggestions for improvement in Series Type

### ISSUE THREAD

**Issue by PN62323424** [Sonarr/Sonarr#5663]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Is your feature request related to a problem? Please describe

sonarr and radarr have solved more than 90% of my automated downloading problems and I really like it. Only one part has never been solved, that is, as long as TVDB and XEM have different definitions for seasons (especially some older anime), it is hard to automate the download of this anime, because if I change the Series Type to anime, then it will trigger hundreds and thousands of searches that will be blocked by many sites. And if I change the Series Type to Standard, because Standard uses TVDB instead of xem data, even if I manually click download, I can only import the first season (which is still a lot of cases).

If I can allow the case that Series Type is anime to search for seasons, it will solve the problem that too many anime searches are blocked.

Or allow anime works with Series Type of Standard to use tvdb instead of xem data, it will solve the problem that only the first season can be imported in Standard mode. (I think this design is not very reasonable, because if you allow anime to search in Standard mode, there should be a way to import all seasons, you can try to download Bleach season 1, 366 episodes, it is not possible to import all episodes automatically in Standard)

Or the third idea, is it possible to allow standard mode to automatically import anime with absolute set names (even if the xem and tvdb seasons of this anime are not classified the same)

I don't know what you think of my idea. Anyway, thanks for your help!

### Describe the solution you'd like

11

### Describe alternatives you've considered

11

**Comment by markus101** [comment]
Both standard and anime types use XEM data, the difference is absolute episode numbers vs season and episode numbers.

Searching anime for seasons is already covered by https://github.com/Sonarr/Sonarr/issues/5616

For the importing issue, please use one of the support channels: [forums](https://forums.sonarr.tv/), [subreddit](https://www.reddit.com/r/sonarr/), [discord ](https://discord.gg/Ex7FmFK), or [IRC](https://web.libera.chat/?channels=#sonarr) for support.
